playwright-junit/src/test/java/com/microsoft/playwright/junit/BrowserDefaultFactoryTests.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.Browser;
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@UseBrowserFactory
+public class BrowserDefaultFactoryTests {
```

Comment:
As we rely on ThreadLocal, can we also add a test for the fixture running on 2 threads concurrently?